pandas/core/apply.py

```diff
@@ -408,6 +421,70 @@ def agg_list_like(self) -> DataFrame | Series:
             )
             return concatenated.reindex(full_ordered_index, copy=False)
 
+    def new_list_like(self, method: str) -> DataFrame | Series:
```

Comment:
can this be something other than `new`?  i.e. a name that will still be helpful/accurate in 12 months

Reply:
Agreed - splitting this up, some discussion in the first part: #43736. "future" has been suggested as a replacement for "new".